packages/utils/upgrade/src/modules/codemod-runner/index.ts

```diff
@@ -0,0 +1 @@
+export { codemodRunnerFactory } from './codemod-runner';
```

Comment:
We can also export types in a CodemodRunner namespace (the callback one can be useful for instance)